### DISCUSSION TITLE
What are good scores for Next.js Custom Metrics?

### DISCUSSION THREAD

**Discussion by jackleslie** [vercel/next.js#22575] [Help]
# Background

For Web Vitals, Google provides information about what constitutes a "good score" for a particular metric. For example, for Largest Contentful Paint (LCP) the scoring chart is:

- Good: < 2.5s
- Needs improvement: > 2.5s & < 4s
- Poor: > 4s

(Source: https://web.dev/lcp)

# Question

What do the scoring charts look like for Next.js Custom Metrics?

**Comment by leerob** [comment]
I agree that providing guidance similar to Web Vitals would be helpful 👍
